setup.cfg

```diff
@@ -32,7 +32,7 @@ package_dir =
 include_package_data = True
 install_requires =
     cftime
-    numpy
+    numpy<1.25.0
```

Comment:
@mikapfl I pinned numpy to get around the latest version causing hell for any test which checks for warnings.  I think it is a xarray issue. Not ideal, but should only be until a new xarray version is released which resolved the "cumprod" -> "cumproduct"

Reply:
Sounds reasonable. Reminds me of https://github.com/pik-primap/primap2/issues/130, if xarray was testing more against the pre-release version of numpy, we wouldn't have this problem. But if we were testing more against the pre-release versions of things, we could also fix things in our own libraries, and alert our upstreams to changes/incompatibilities.

Also, kind of a pointless change from numpy, why not keep the old name forever?